doc/admin-api.md

```diff
@@ -710,3 +710,15 @@ Content-Type: text/plain
 ```
 
 [Back to TOC](#Table-of-Contents)
+
+*API*：/apisix/admin/plugins/?all=true
```

Comment:
valid to use this format?

cc @membphis @spacewander @gxthrj

Reply:
![image](https://user-images.githubusercontent.com/30819887/103256692-1f75cc80-49c9-11eb-94ae-29cfcaeab1ad.png)
I saw that after this format was proposed on the mail-list: https://lists.apache.org/thread.html/r33b8d142ba57cfda467b4912ec94d5ae9c77b1eec4982f39f2e6bf0e%40%3Cdev.apisix.apache.org%3E, no one ended up having a problem with it.

Reply:
![image](https://user-images.githubusercontent.com/2106987/103257299-9c09aa80-49cb-11eb-8969-0f05d8b7199e.png)

But there has a `/` in the API

Reply:
`/apisix/admin/plugins/?all=true` and `/apisix/admin/plugins?all=true`  both can be accessed.

Reply:
should I update this md?

Reply:
Would be better to use `/apisix/admin/plugins?all=true`

Reply:
can reopen this PR?

Reply:
resolve by #3153